packaging/rpm/microshift-images.spec

```diff
@@ -0,0 +1,62 @@
+%global imageStore %{buildroot}/var/lib/microshift/images
+
+Name: microshift-images
+Version: 1
+Release: 1
+# this can be %{timestamp}.git%{short_hash} later for continous main builds
+Summary: Create custom container storage
+License: ASL 2.0
+URL: https://github.com/redhat-et/microshift
+
+BuildRequires:  podman
+BuildRequires: crio
+
+
+%description
+This rpm creates a custom RO container storage and pull images and add path to additional container image stores.
+
+%prep
+
+if [ -d  %{imageStore} ] 
+then
+   sudo rm -rf  %{imageStore}
+fi
+
+
+%build
+
+
+%install
```

Comment:
Shouldn't this belong into the %build section, such that the RPM _contains_ the images (rather than pulling them when installed)? Not sure the ImageBuilder can actually run stuff on podman when building ostrees.

Reply:
As per the doc, https://rpm-packaging-guide.github.io/#what-is-a-spec-file
```
%build: Command or series of commands for actually building the software into machine code (for compiled languages) or byte code (for some interpreted languages).
```

```
%Install: Command or series of commands for copying the desired build artifacts from the %builddir (where the build happens) to the %buildroot directory (which contains the directory structure with the files to be packaged). This usually means copying files from ~/rpmbuild/BUILD to ~/rpmbuild/BUILDROOT and creating the necessary directories in ~/rpmbuild/BUILDROOT. This is only run when creating a package, not when the end-user installs the package. See [Working with SPEC files](https://rpm-packaging-guide.github.io/#working-with-spec-files) for details.
```

Since we are not building s/w and only using rpm to package files, I put it under `%install`

Reply:
it's .spec files just being confusing,

What you expect for install is %post

%install is the phase that takes the %build, and put's it into the fake root filesystem,
from where %file will gather stuff.